### PR TITLE
Retrieve tags from tags endpoint

### PR DESCRIPTION
**Pull request by Raph22**

### CODE REVIEW

File: src/Api.ts [hunk 1]
```diff
@@ -51,7 +58,7 @@ export default class Api {
    * then cached).
    */
   get(cb?: RequestCallback<ResolvedApi>): Promise<ResolvedApi> {
-    return this.httpClient.cachedRequest<ApiData>(this.url, { ttl: this.apiDataTTL }).then((data) => {
+    return this.httpClient.cachedRequest<ApiData>(`${this.originalUrl}/tags`, { ttl: this.apiDataTTL }).then((data) => {
```
Comment (srenault) [inline src/Api.ts:61]: to remove

File: src/Api.ts [hunk 2]
```diff
@@ -19,21 +19,28 @@ function separator(url: string): string {
...
       this.options.routes && `routes=${encodeURIComponent(JSON.stringify(this.options.routes))}`
     ]
     .filter(Boolean)
 
     if(queryStrings.length > 0) {
-      this.url += separator(url) + queryStrings.join('&')
+      this.url += separator(url) + queryStrings.join('&');
+    }
+
+    if(accessTokenParam) {
```
Comment (srenault) [inline src/Api.ts:42]: to remove

File: src/Api.ts [hunk 3]
```diff
@@ -60,4 +67,23 @@ export default class Api {
       throw error;
     });
   }
+
+  getTags(cb?: RequestCallback<Array<string>>): Promise<Array<string>> {
```
Comment (srenault) [inline src/Api.ts:71]: I think we shouldn't callback anymore.

File: src/client.ts [hunk 4]
```diff
@@ -76,4 +80,9 @@ export class DefaultClient implements Client {
     const api = new Api(url, options);
     return api.get();
   }
+
+  static getTags(url: string, options?: ApiOptions): Promise<Array<string>> {
```
Comment (srenault) [inline src/client.ts:84]: I think we don't need this static function

File: src/index.ts [hunk 5]
```diff
@@ -27,3 +29,11 @@ function getApi(url: string, options?: ApiOptions): Promise<ResolvedApi> {
 function api(url: string, options?: ApiOptions): Promise<ResolvedApi> {
   return getApi(url, options);
 }
+
+function getTags(url: string, options?: ApiOptions): Promise<Array<string>> {
```
Comment (srenault) [inline src/index.ts:33]: I think we don't need this as well

**Comment by Raph22** [comment]
Replaced by #162 by using form

**Pull request closed without merging**